src/version.js

```diff
@@ -1 +1 @@
-export default '3.0.1';
+export default '4.0.0-rc4';
```

Comment:
- [ ] Create an issue to inject version as part of build process to `index.html` config object.

Reply:
👀